{"name": "test_login_user_invalid_data_login_error[-skoritsey]", "status": "passed", "steps": [{"name": "Login with wrong email: ", "status": "passed", "start": 1725198361688, "stop": 1725198361951}], "parameters": [{"name": "email", "value": "''"}, {"name": "password", "value": "'skoritsey'"}], "start": 1725198361311, "stop": 1725198361952, "uuid": "88dfe492-77c5-47a7-b65d-8d1ae32878a2", "historyId": "379db316b825629d2b63822922bcb265", "testCaseId": "dffb61152879fb5542bb13eac9dd1ece", "fullName": "tests.test_login_user.TestLoginUser#test_login_user_invalid_data_login_error", "labels": [{"name": "parentSuite", "value": "tests"}, {"name": "suite", "value": "test_login_user"}, {"name": "subSuite", "value": "TestLoginUser"}, {"name": "host", "value": "MacBook-Air-Elizaveta.local"}, {"name": "thread", "value": "84379-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "tests.test_login_user"}], "links": [{"type": "tms", "url": "Login user with invalid data", "name": "Login user with invalid data"}]}